src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,10 @@
+package calculator;
+
+public class Calculator {
```

Comment:
This is not the intended approach.
Please remove this file.
Instead use `com.epam.tat.module4.Calculator` from `calculator-1.0.jar` (required dependency)